src/use/keyboard.ts

```diff
@@ -0,0 +1,15 @@
+import { useEventListener } from '@vueuse/core';
+import { Key } from 'ts-key-enum';
+
+type EventType = 'keyup' | 'keydown';
+type Bindings = {
+  [k in Key]?: () => void;
+};
+
+export function useKeyboard(bindings: Bindings, eventType: EventType = 'keyup') {
+  useEventListener(eventType, (event: KeyboardEvent) => {
+    if (bindings[event.key] != null) {
+      bindings[event.key]();
+    }
```

Comment:
`useEventListener` helps a lot for these things. It registers on mounted and unregisters on destroy.